src/prefect/server/api/server.py

```diff
@@ -315,7 +315,7 @@ def ui_settings():
         and not ephemeral
     ):
         ui_app.mount(
-            "/",
+            os.environ.get("PREFECT_UI_SERVE_BASE", "/"),
```

Comment:
Do we want this to be a setting like `PREFECT_UI_API_URL` so you can configure it in your profile or as an environment variable?

Reply:
Hm I suppose we could do that!

Reply:
This has been updated @desertaxle